polling_stations/apps/data_collection/management/commands/import_west_lancashire.py

```diff
@@ -67,20 +62,32 @@ def address_record_to_dict(self, record):
             "100010684612",  # WN80AG -> WN80AH : 7 Ormskirk Road, Up Holland, Skelmersdale
             "100012654494",  # L397HS -> L397HQ : Rose Croft, Broad Lane, Downholland, Ormskirk
             "100012821133",  # WN88AU -> WN80PB : 21 Liverpool Road, Skelmersdale
-            "10012363326",  # L392EJ -> L392DE : 1000 Walmsley Drive, Ormskirk
-            "100010675944",  # WN88PU -> WN88EN : 83B School Lane, Skelmersdale
             "100012657542",  # L407SN -> L407SL : The Homestead Merridale Farm, High Lane, Ormskirk
             "100012415199",  # L407SN -> L407SL : The Mount, High Lane, Ormskirk
-            "10012341150",  # L407SN -> L407GA : 6 Blythe Meadow, High Lane, Ormskirk
-            "10012341151",  # L407SN -> L407GA : 7 Blythe Meadow, High Lane, Ormskirk
             "10012356919",  # WN80HS -> WN89QS : 93 Dearden Way, Up Holland, Skelmersdale
             "10012344656",  # L406JA -> L406JT : Ottershead Barn, Dicks Lane, Lathom, Ormskirk
             "100012414799",  # L406JA -> L406JT : Whitestone Cottage, Dicks Lane, Lathom, Ormskirk
             "100012418034",  # WN69EN -> WN69EQ : Glen View, Hall Lane, Wrightington, Wigan
             "100010677297",  # WN80EW -> WN80EN : Hallbridge Farmhouse, 59 Dingle Road, Up Holland, Skelmersdale
             "100010682656",  # WN80QY -> WN80QZ : Pearsons Farm, Lafford Lane, Up Holland, Skelmersdale
-            "100010667268",  # L397EA -> L394QW : 245 St Helens Road, Lathom Skelmersdale
+            "100012657092",  # L405SW -> L405UZ : Lyme House, Junction Lane, Burscough, Ormskirk
+            "10012357601",  # L393BH -> L393BW : 14 Park Road, Ormskirk
+            # Just not sure
+            "10012354202",  # L404BX -> L404BF : Antioch, Wheat Lane, Lathom, Ormskirk
+            "10012358235",  # L404BX -> L404BF : El-Cyd Houseboat, Wheat Lane, Lathom, Ormskirk
+        ]:
+            rec["accept_suggestion"] = False
+
+        if uprn in [
+            "100012416134",  # L390HF -> L390HP : The Flat Quattros, Rainford Road, Bickerstaffe, Ormskirk
+            "10012363368",  # WN89HU -> WN89BH : 1 Beechtrees Mews, Beechtrees, Skelmersdale
+            "10012363370",  # WN89HU -> WN89BH : 3 Beechtrees Mews, Beechtrees, Skelmersdale
+            "10012363372",  # WN89HU -> WN89BH : 5 Beechtrees Mews, Beechtrees, Skelmersdale
+            "10012363374",  # WN89HU -> WN89BH : 7 Beechtrees Mews, Beechtrees, Skelmersdale
+            "10012363376",  # WN89HU -> WN89BH : 9 Beechtrees Mews, Beechtrees, Skelmersdale
+            "10012363378",  # WN89HU -> WN89BH : 11 Beechtrees Mews, Beechtrees, Skelmersdale
         ]:
+            # Wrong postcode in AddressBase
             rec["accept_suggestion"] = False
```

Comment:
These can be included in the list above.

Reply:
Done :-)